### PR TITLE
Adding labels.yaml for check-labels munger.

### PR DESCRIPTION
**Pull request by foxish**
List of labels to be created in the contrib repository.
The list may need to be pruned, in case some labels do not make sense for contrib.

### CODE REVIEW

**Comment by eparis** [comment]
is alphabetical sort possible to help review? There are a bunch we don't need/want...

**Comment by foxish** [comment]
@eparis Done. PTAL.

**Comment by foxish** [comment]
cc @bgrant0607

File: labels.yaml [hunk 1]
```diff
@@ -0,0 +1,285 @@
+labels:
+  - name: 0 - Backlog
```
Comment (bgrant0607) [inline labels.yaml]: Not needed

File: labels.yaml [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+    color: CCCCCC
+  - name: 0 - Triage
```
Comment (bgrant0607) [inline labels.yaml]: Not needed

File: labels.yaml [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+    color: AAAAAA
+  - name: 1 - Ready
```
Comment (bgrant0607) [inline labels.yaml]: Not needed

File: labels.yaml [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+    color: CCCCCC
+  - name: 2 - Working <= 5
```
Comment (bgrant0607) [inline labels.yaml]: Not needed

File: labels.yaml [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+    color: CCCCCC
+  - name: 3 - Review
+    color: CCCCCC
+  - name: 4 - Done
+    color: CCCCCC
+  - name: '[deprecated--DO NOT USE]old-team/master'
```
Comment (bgrant0607) [inline labels.yaml]: Not needed.

File: labels.yaml [hunk 6]
```diff
@@ -0,0 +1,285 @@
...
+    color: e11d21
+  - name: documentation/better-examples
+    color: f9d0c4
+  - name: documentation/confusing
+    color: f9d0c4
+  - name: documentation/content-gap
+    color: f9d0c4
+  - name: documentation/out-of-date
+    color: f9d0c4
+  - name: e2e-not-required [deprecated use retest-not-required]
```
Comment (bgrant0607) [inline labels.yaml]: Not needed.

File: labels.yaml [hunk 7]
```diff
@@ -0,0 +1,285 @@
...
+    color: c7def8
+  - name: kind/flake
+    color: f7c6c7
+  - name: kind/friction
+    color: c7def8
+  - name: kind/mesos-flake
+    color: f7c6c7
+  - name: kind/new-api
+    color: c7def8
+  - name: kind/old-docs
```
Comment (bgrant0607) [inline labels.yaml]: Not needed.

File: labels.yaml [hunk 8]
```diff
@@ -0,0 +1,285 @@
...
+    color: 0052cc
+  - name: component/kubelet
+    color: 0052cc
+  - name: component/node-e2e
+    color: 0052cc
+  - name: component/nodecontroller
+    color: 0052cc
+  - name: component/ui
+    color: 0052cc
+  - name: davidopp
```
Comment (bgrant0607) [inline labels.yaml]: Not needed.

**Comment by bgrant0607** [comment]
I don't think it's that important to prune this list. I commented on a few -- deprecated labels, etc. 

LGTM after those are removed.

**Comment by eparis** [comment]
I wondered if we should come up with a 'standard' list. One without all of the `areas`. Maybe even without all of the `teams`. These don't hurt. But they don't help either...

**Comment by bgrant0607** [comment]
@eparis 

It would be helpful to identify labels upon which the bot actually depends.

I do hope we can use the issue classifier on multiple repos, but I agree not all labels make sense for all repos.

I'd actually like to rename many of the labels (e.g., team and component labels) before we copy them to lots of repos.

**Comment by foxish** [comment]
@bgrant0607 Done!

**Comment by bgrant0607** [comment]
LGTM

**Pull request merged**